test_cases/DrugCentral_creative.json

```diff
@@ -50,5 +50,7 @@
     "test_case_predicate_name": null,
     "test_case_predicate_id": null,
     "test_case_input_id": null,
-    "test_case_runner_settings": []
```

Comment:
I had opened https://github.com/NCATSTranslator/Tests/pull/9 which should handle running a subset of benchmarks while we work on schema updates.

Reply:
```
    "test_case_objective": "QuantitativeTest",
    "test_case_source": null,
    "test_case_predicate_name": null,
    "test_case_predicate_id": null,
    "test_case_input_id": null,
    "test_case_runner_settings": [
        "limit_queries"
    ]
}
```

Reply:
It doesn't really matter, but I don't think we need the runner settings for this benchmark. It's already a subset in the benchmarks repo.